POI Details: White corners Gone when image is fully collapsed

sebastianlazea opened the pull request:
Poi Screen: Capitalized categories and underscore Gone

sebastianlazea committed:
POI Details: White corners Gone when image is fully collapsed
Poi Screen: Capitalized categories and underscore Gone

teodorga reviewed (changes requested):
-

teodorga reviewed (changes requested):
Changes on white corners

sebastianlazea committed:
POI Details: White corners Gone when image is fully collapsed (final version)
                                                      Poi Screen: Capitalized categories and underscore Gone

sebastianlazea committed:
POI Details Screen: Removed Status Bar

sebastianlazea committed:
POI Details Screen: Removed Status Bar (transparent now)

sebastianlazea committed:
POI Details Screen: Removed Status Bar (transparent now)

sebastianlazea committed:
-POI Screen: added Margin Top to city name
-Poi Details Screen: Invisible placeholders for destination and address before data is populated

sebastianlazea committed:
Merge branch 'develop' into ts_13_poidetailsscreen
